examples/atlas-search.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/**
+ * This example demonstrates how to create a search index and perform a text search.
+ * It requires a MongoDB Atlas M10+ cluster with Sample Dataset loaded.
+ */
+
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use Closure;
+use MongoDB\Client;
+use RuntimeException;
+
+use function define;
+use function getenv;
+use function hrtime;
+use function iterator_to_array;
+use function preg_match;
+use function sleep;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI');
+if (! $uri || ! preg_match('/\.(mongodb\.net|mongodb-dev\.net)/', $uri)) {
```

Comment:
Users would not be running on `mongodb-dev.net`. Should we remove this and stick with `strpos()` on "mongodb.net" (like the original unified test runner)?

Reply:
That's the only way I can run this test. I don't have the privilege of running an M10 cluster on Atlas, but I can run one on cloud-dev.

Reply:
Ok, I can do this: `MONGODB_URI="mongodb+srv://user:*****@cluster0.****.mongodb-dev.net/?hehe=.mongodb.net`